pyproject.toml

```diff
@@ -6,5 +6,12 @@ authors = ["Version One"]
 license = "MIT License"
 
 [tool.black]
-line-length = 79
+line-length = 120
 experimental-string-processing = 1
+
+[tool.flake8]
+per-file-ignores = [
+    './app/assess/routes.py:F405,F403,E731',
```

Comment:
Could we add a comment about what is ignored here and why please?

Reply:
Actually, this setting is moved from `.pre-commit-config.yaml`. But now this file no more exists so removing the settings